contrib/devtools/README.md

```diff
@@ -144,6 +144,13 @@ Configuring the github-merge tool for the bitcoin repository is done in the foll
     git config githubmerge.testcmd "make -j4 check" (adapt to whatever you want to use for testing)
     git config --global user.signingkey mykeyid (if you want to GPG sign)
 
+Create and verify timestamps of merge commits
+---------------------------------------------
+To create or verify timestamps on the merge commits, download the OpenTimestamps
```

Comment:
Maybe link to OTS releases?

Reply:
Yeah, this instruction is a bit misleading: it's not just enough to download that script, you also have to install OpenTimestamps. The easiest way to do that is simply run:

    pip3 install opentimestamps-client

Reply:
I'd rather link to the "manual" installation, since it seems common to typosquat package names for pip.

Reply:
I don't think that's much of a risk, given that we'd be providing copy-n-pastable instructions right there.